src/WebJobs.Script/Eventing/Rpc/RpcWebhostChannelReadyEvent.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.Eventing
 {
-    internal class RpcChannelReadyEvent : RpcChannelEvent
+    internal class RpcWebhostChannelReadyEvent : RpcChannelEvent
     {
-        internal RpcChannelReadyEvent(string id, string language, ILanguageWorkerChannel languageWorkerChannel,
+        internal RpcWebhostChannelReadyEvent(string id, string language, ILanguageWorkerChannel languageWorkerChannel,
```

Comment:
nit: Naming/casing would be `RpcWebHostChannelReadyEvent`, also, can you add a quick type summary here documenting that this is the Application level scope?